public/src/node.js

```diff
@@ -448,9 +448,9 @@ export default class Node {
             }
         }
         var colorNode = colors['stroke'];
-        const colorNodeConnect = colors['color_wire_con']
-        const colorNodePow = colors['color_wire_pow']
-        const colorNodeLose = colors['color_wire_lose']
```

Comment:
You just changed the color of all the nodes to one color! I think it looks ugly. We need to give different colors depending on the value. Can you check old ui colors?

Reply:
Ok I'll revert back to this